### PR TITLE
Fix Snowflake max column length tests

### PR DESCRIPTION
**Pull request by chenjian2664**
## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

**Review by ebyhr — DISMISSED** [review]
This change doesn't work. I would recommend running tests locally before sending PR.

**Comment by ebyhr** [comment]
The current change still doesn't work. Some tests (e.g. `testAlterTableRenameColumnToLongName`, `testAlterTableAddLongColumnName` & `testCreateTableWithLongColumnName`) will fail. 

I would recommend running all tests locally.

File: plugin/trino-snowflake/src/test/java/io/trino/plugin/snowflake/TestSnowflakeConnectorTest.java [hunk 1]
```diff
@@ -233,28 +231,6 @@ public void testDeleteWithLike()
                 .hasStackTraceContaining("TrinoException: " + MODIFYING_ROWS_MESSAGE);
     }
 
-    @Test
-    @Override
-    public void testCreateTableWithLongColumnName()
```
Comment (findepi) [inline plugin/trino-snowflake/src/test/java/io/trino/plugin/snowflake/TestSnowflakeConnectorTest.java:238]: unrelated
